Examples/Io/Root/src/RootMeasurementWriter.cpp

```diff
@@ -26,9 +29,186 @@ namespace Acts {
 class Surface;
 }  // namespace Acts
 
-ActsExamples::RootMeasurementWriter::RootMeasurementWriter(
-    const ActsExamples::RootMeasurementWriter::Config& config,
-    Acts::Logging::Level level)
+namespace ActsExamples {
+
+struct RootMeasurementWriter::DigitizationTree {
```

Comment:
I'm assuming this is mostly unchanged?

Reply:
Right the tree handling itself did not change significantly